### PR TITLE
Add preliminary generator

### PR DESCRIPTION
**Pull request by sarayourfriend**
To test:
1. Install ruby
2. cd into the `generator` directory
3. run `gem install bundle jekyll`
4. run `bundle exec jekyll serve`

Check the site out at `localhost:4000`

### CODE REVIEW

**Comment by sarayourfriend** [comment]
Adding note to self to move the resource files into separate folders per city. Add a `shared` folder. Remove the city slugs as a list and change it to either be a single city slug OR `'*'` when it should be added to all cities (like Trans Lifeline). Also investigate if there's a way to pull the city slug from the folder name (I think this should be possible based on some of what I've seen dumped out by the `inspect` filter).

**Comment by sarayourfriend** [comment]
Update: I was able to infer the city slug from the path name but not the category. It's easy to check if a string contains something in liquid (the template language Jekyll uses) but not to parse a full string and try to pull out just a section of it, even with some bizzare combination of `where_exp` and `split`s

File: generator/Gemfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+source "https://rubygems.org"
+# Hello! This is where you manage which Jekyll version is used to run.
+# When you want to use a different version, change it below, save the
+# file and run `bundle install`. Run Jekyll with `bundle exec`, like so:
+#
+#     bundle exec jekyll serve
+#
+# This will help ensure the proper Jekyll version is running.
+# Happy Jekylling!
+gem "jekyll", "~> 4.2.0"
```
Comment (AetherUnbound) [inline generator/Gemfile:10]: 🌱

**Review by AetherUnbound — APPROVED** [review]
This looks great! I think I can mostly follow how the templates are being composed and how resources are compiled. It would certainly be easy for me to add more, not knowing much else about the project 💯

**Pull request merged**